clowder/utility/git_submodule_utilities.py

```diff
@@ -95,27 +89,25 @@ def _create_repo_herd_branch(self, url, remote, branch, default_ref, depth=0):
 
     def _submodules_clean(self):
         """Clean all submodules"""
+        self._submodule_command('foreach', '--recursive', 'git', 'clean', '-ffdx',
+                                error_msg=' - Failed to clean submodules')
+
+    def _submodule_command(self, *args, error_msg=' - submodule command failed'):
```

Comment:
it looks like for some reason Python 2 doesn't seem to like this bit `error_msg=' - submodule command failed'`

Reply:
Yeah pylint didn't like it either, can't figure out what's the issue.

Reply:
you might have to swap the order of `*args` and `error_msg` for Python 2 to handle it https://stackoverflow.com/questions/9872824/calling-a-python-function-with-args-kwargs-and-optional-default-arguments